speechbrain/lobes/models/transformer/TransformerASR.py

```diff
@@ -262,7 +263,7 @@ def __init__(
         conformer_activation: Optional[nn.Module] = Swish,
         branchformer_activation: Optional[nn.Module] = nn.GELU,
         attention_type: Optional[str] = "regularMHA",
-        max_length: Optional[int] = 2500,
+        max_length: Optional[int] = 10000,
```

Comment:
This breaks backward compatibility. I 100% agree that 2500 is too little, but it will break backward compatibility when loading old checkpoints... can we keep it to 2500?